plugins/window_size/lib/src/window_size_channel.dart

```diff
@@ -73,35 +102,21 @@ class WindowSizeChannel {
 
   /// Returns a list of screens.
   Future<List<Screen>> getScreenList() async {
-    try {
-      final screenList = <Screen>[];
-      final response =
-          await _platformChannel.invokeMethod(_getScreenListMethod);
-
-      for (final screenInfo in response) {
-        screenList.add(_screenFromInfoMap(screenInfo));
-      }
-      return screenList;
-    } on PlatformException catch (e) {
```

Comment:
Why do you need to remove the try catch? Is there no possibility for platform exceptions anymore?

Reply:
See my comments earlier in the review; I don't think we ever should have been suppressing these exceptions.